hostscripts/rpm-packaging/createproject.py

```diff
@@ -37,8 +37,8 @@
 def pymodule2pkg(spectemplate):
     specname = os.path.splitext(spectemplate)[0]
     modulename = os.path.splitext(os.path.basename(specname))[0]
-    pkgname = pymod2pkg.module2package(modulename,
-                                       platform.linux_distribution()[0])
+    pkgname = pymod2pkg.module2package(
+        modulename, platform.linux_distribution()[0] or 'suse')
```

Comment:
You could use the `distro` package.

Reply:
Yes, I could.

Reply:
Then I don't understand the point of the test. If we already know that the outcome is suse, why use a module?